src/components/Organisms/MarketingPreferencesDS/_TextInput.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useFormContext } from 'react-hook-form';
+import Input from '../../Atoms/Input/Input';
+
+const TextInput = ({
+  fieldName, label,
+  isRequired, fieldType, ...rest
+}) => {
+  const { errors, register } = useFormContext();
+
+  const props = {
+    name: fieldName,
+    type: fieldType,
+    label,
+    placeholder: label,
+    errorMsg: errors && errors[fieldName]
+      && errors[fieldName].message,
+    isRequired,
+    'aria-required': isRequired,
+    labelProps: {
+      className: errors && errors[fieldName]
+      && errors[fieldName].message ? 'error-message' : ''
```

Comment:
I can't seem to see any styling associated with this class, is it needed?

Reply:
I think this might be some vestigial code from a previous design choice, will strip it owtttt